res/xml-v29/preferences_appearance.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i am not really happy about this duplication, we should at least add a bit WARNING atop of both files.

however, for now: the duplication is only needed because of the changed default value, or?
i suggest to just leave the default to "light" in all cases - we seem to have some issues with dark-mode on some systems, so we should not force using that until we really know it is fixed.

Reply:
Issue  #1748 is that the system is in dark mode and DC is in light mode. As soon as the user switched DC to dark mode, everything was fine. So, always following the system mode would definitely have prevented this special bug (that is, at least until the user changes the setting from "system default" to "light")

Or are you talking about other issues with the dark mode?

Reply:
yip, you're right, so, changing default should not worsening things.

but i would still suggest to add a hint atop of the duplicated files. otherwise if will be forgotten on the next option added/changed.

> Or are you talking about other issues with the dark mode?

no, i do not know about other issues with dark mode.